src/app.js

```diff
@@ -8,8 +8,17 @@ var DummyLesson = require("./lesson/dummy_lesson");
 var Lesson01 = require("./lesson/lesson01");
 var LessonEnvironment = require("./view/lesson_environment");
 
-var startLesson = function(lessonClass, domElement) {
-  ReactDOM.render(<LessonEnvironment lesson={new lessonClass()} />,
+var startLesson = function(lesson, domElement) {
+  var lessonInstance = null;
+  if (typeof(lessonClass) === 'function') {
```

Comment:
Is this right? I don't see `lessonClass` defined anywhere. I am assuming it should be passed as a parameter.